common-setup/profile

```diff
@@ -9,9 +9,19 @@
 # cannot be in the profile file, which is read later on
 # Otherwise, we will for instance get "millis: command not found" on
 # every new terminal, even though it is found on the CLI when testing
-if [ -d ~/bin ] ; then
-    PATH=~/bin:"${PATH}"
-fi
+_paths=(
```

Comment:
Unfortunately, this is not working. Profile is read before .bashrc and its aliases